yum-requirements.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# List of packages installed with yum on CentOS 7 platforms.
+#
+# This list of packages is also included in the documentation at
+# doc/ug/install_instructions/index.md. When updating this file also check if
+# doc/ug/install_instructions/index.md needs to be updated as well.
+#
+# Keep it sorted.
+autoconf
+bison
+build-essential
+clang-format
+curl
+doxygen
+epel-release
```

Comment:
I don't think that package itself does the trick? You'll need to enable another repository first; this should go into the documentation itself, not this file.

Reply:
I don't think so either. But on the CentOS 7 machine I have access to, this package is there as well. However, it could be that by enabling the EPEL release (I've added a second reference in the documentation) the `epel-release` package shows up as installed. Unfortunately, I can't do any testing here as I don't have root access to this machine. It would be great if @svenka3 could clarify this.

Reply:
The instructions you linked to on enabling the EPEL repository already include a step to install this package, you can safely remove it from here. (There are other ways to enable this repository or similar repositories which provide the same packages.)

Reply:
Sounds good. I have removed it. We can always re-add it later if it's really necessary.